src/atomate2/vasp/schemas/calculation.py

```diff
@@ -493,7 +497,8 @@ def from_vasp_outputs(
         outcar_dict = outcar.as_dict()
         outcar_dict.pop("run_stats")
 
-        structure = vasprun.final_structure
+        # structure = vasprun.final_structure
```

Comment:
Can you change this line to:
```suggestion
        # use structure from CONTCAR as it is written to greater precision than in the vasprun
```